tensorboard/webapp/feature_flag/store/feature_flag_metadata.ts

```diff
@@ -111,6 +111,11 @@ export const FeatureFlagMetadataMap: FeatureFlagMetadataMapType<FeatureFlags> =
       queryParamOverride: 'showFlags',
       parseValue: parseBoolean,
     },
+    allowRangeSelection: {
+      defaultValue: false,
+      queryParamOverride: 'allowRange',
```

Comment:
It would be nice to keep the query param overrides as close the flag name as possible.
```suggestion
      queryParamOverride: 'allowRangeSelection',
```